backend/x11/backend.c

```diff
@@ -44,7 +44,7 @@ static uint32_t xcb_button_to_wl(uint32_t button) {
 static bool handle_x11_event(struct wlr_x11_backend *x11, xcb_generic_event_t *event) {
 	struct wlr_x11_output *output = &x11->output;
 
-	switch (event->response_type) {
+	switch (XCB_EVENT_RESPONSE_TYPE(event)) {
```

Comment:
Not my call (I have nothing to do with wlroots), but I'd say that this change does not belong into this commit. The commit message is about atoms.